### PR TITLE
Add version on Header

### PR DESCRIPTION
**Pull request by joao-alberto**
Ref #113 

| Desktop  |
|:-:|
| screen shot 2018-07-27 at 15 32 11  | 

| Mobile  |
|:-:|
| screen shot 2018-07-27 at 20 07 11 |

### CODE REVIEW

**Comment by diegohaz** [comment]
Awesome, @joao-alberto. Can we try to add it to the mobile sidebar?

**Comment by wordofchristian** [comment]
Yeah, this information seems way less important on mobile. I think it would be find to deprioritize it's prominence.

**Comment by wordofchristian** [comment]
This is probably out of scope for this PR, but it might also be useful to display (latest) or (current) after the version number if this is the latest/current version.

That information would have to be sourced in a completely different way though. Perhaps it could be based on git tag and determined at compile time.

**Comment by diegohaz** [comment]
@wordofchristian I was thinking about something like that when we have v1, v2 etc. 

For now, I wanted to transform the version text into a link that routes to CHANGELOG.md, but it can be done in another PR.

File: website/src/utils/getRelease.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { repository, version as pkgVersion } from "../../../package.json"
+
+const version = `v${pkgVersion}`;
+
+const url = () => `https://github.com/${repository}/releases/tag/${version}`;
```
Comment (diegohaz) [inline website/src/utils/getRelease.js]: Let's use `https://github.com/${repository}/blob/master/CHANGELOG.md`.

Also, could you run `npm run lint -- --fix`?

**Comment by Thomazella** [comment]
screen shot 2018-07-27 at 9 35 42 pm

🤔 I think it looks a bit detached from the other elements in the sidebar. I'll edit some photoshop alternatives here for us.

File: website/src/utils/getRelease.js (hunk 1 above)
Comment (joao-alberto) [reply website/src/utils/getRelease.js]: Oops, sure!

**Comment by Thomazella** [comment]
I prefer the `Reakit v0.12.0` on top, same alignment.

**Comment by diegohaz** [comment]
Let's keep the [original one](https://user-images.githubusercontent.com/10820159/43350107-f6fff7ba-91da-11e8-8c54-2830d2d162c6.png) (actually, that's the one I liked the most XD). We can tweak styles later.

**Comment by joao-alberto** [comment]
@diegohaz @Thomazella 

- [x] Lint
- [x] Link to changelog instead 'releases'

**Pull request merged**